.eslintrc.json

```diff
@@ -16,7 +16,8 @@
   "globals": {
     "__SERVER__": true,
     "__CLIENT__": true,
-    "__PRODUCTION__": true
+    "__PRODUCTION__": true,
+    "jasmine": true
```

Comment:
Not sure if Jasmine should be in the eslintrc file, since it's not a global when running production or development versions.